flang/runtime/extensions.cpp

```diff
@@ -37,5 +77,30 @@ void FORTRAN_PROCEDURE_NAME(getarg)(
   (void)RTNAME(GetCommandArgument)(
       n, &value, nullptr, nullptr, __FILE__, __LINE__);
 }
+
+// CALL GETLOG(USRNAME)
+void FORTRAN_PROCEDURE_NAME(getlog)(std::int8_t *arg, std::int64_t length) {
```

Comment:
nit: I think we should use `std::byte` instead of `std::int8_t`, while they're functionally going to be the same here since you're just casting it around it would be more clear that that's the intention for the type